extensions/amp-video/0.1/amp-video.md

```diff
@@ -156,6 +156,13 @@ Indicates the album/collection the video was taken from, specified as a string.
 
 Indicates the name/title of the video, specified as a string. If not provided, the Media Session API helper uses either the `aria-label` attribute or falls back to the page's title.
 
+[filter formats="stories"]
+
+#### cache
```

Comment:
Unless we document how to implement your own system, we should keep this specific to the Google provided video caching system